tony-core/src/main/java/com/linkedin/tony/TonyPolicyProvider.java

```diff
@@ -4,16 +4,20 @@
  */
 package com.linkedin.tony;
 
+import com.linkedin.tony.rpc.MetricsRpc;
+import com.linkedin.tony.rpc.TensorFlowCluster;
 import org.apache.hadoop.security.authorize.PolicyProvider;
 import org.apache.hadoop.security.authorize.Service;
-import com.linkedin.tony.rpc.TensorFlowCluster;
 
 /**
  * PolicyProvider for Client to AM protocol.
  **/
-public class TFPolicyProvider extends PolicyProvider {
+public class TonyPolicyProvider extends PolicyProvider {
```

Comment:
Thanks for the clean up! Could you write a doc on how to add a new RPC between AM & Executors?

Reply:
Sure, I'll add a Wiki page for this on the TonY GitHub Wiki.

Reply:
Created https://github.com/linkedin/TonY/wiki/Adding-an-RPC-client---server-pair-between-client-and-AM